### DISCUSSION TITLE
Create upgraded textures for some items, blocks and tiles

### DISCUSSION THREAD

**Discussion by DevWolf31** [fnuecke/oc2#233] [Ideas]
The current textures of the blocks, items and tiles are not very good, what can we say... They should be improved, right? I'm not talking about ``128 x 128`` textures but rather simple ``16 x 16`` textures that are more attractive to users.
<br>
English is not the language I am most fluent in. Sorry if I was wrong about something.

**Comment by DevWolf31** [comment]
I can do the textures, if you want

**Comment by atirutw** [comment]
The textures are *okay* IMO. Original OC mod's textures are way worse especially with the gradients :skull:.

Well, you still could try and see if you can improve upon OC2's textures. I think the devs will appreciate it if you can improve them.

> English is not the language I am most fluent in. Sorry if I was wrong about something.

Nobody would ever notice about that, at least in this post.

**Reply by DevWolf31** [reply]
What I do think should be improved is the cable model, it looks very strange...

**Reply by DevWolf31** [reply]
I'm doing the new textures right now

**Comment by DevWolf31** [comment]
Here is a preview of the back side of the computer: <br> ![computer-back](https://user-images.githubusercontent.com/88561321/186270086-08e9358b-3f45-420c-b269-795248e6977c.png)
<br>
Front side of the computer: <br> 
![computer-front](https://user-images.githubusercontent.com/88561321/186494218-2aaf71c2-e3cd-44d2-8de3-915169532cb3.png)
<br>
Preview of the network hub:
![hub](https://user-images.githubusercontent.com/88561321/186494375-a17b3701-a3a5-4ef7-8c28-3f4fa72cb071.png)
<br>
I chenged the network connector colors to match with the network hub: <br> 
![connector](https://user-images.githubusercontent.com/88561321/186504360-75647f36-6fb4-489d-ab2f-e36c16d45be5.png)
<br>
Say me what do you think.

**Reply by atirutw** [reply]
Looks really good! How's the front side?

**Comment by DevWolf31** [comment]
I updated the comment, to show you the progress

**Comment by untodesu** [comment]
I'd also love to see actual separate serial terminals with cool models and stuff instead of framebuffers that feel sluggish...
